.gitignore

```diff
@@ -0,0 +1,5 @@
+# Dependencies
```

Comment:
Might be better to use a boilerplate gitignore. For example .DS_file is not ignored.